### PR TITLE
Update Deutsch locale

### PR DESCRIPTION
**Pull request by TooWaBoo**

### CODE REVIEW

File: rtdata/languages/Deutsch [hunk 1]
```diff
@@ -967,16 +987,14 @@ MAIN_TAB_RAW;RAW
...
-MAIN_TAB_WAVELET_TOOLTIP;Taste: <b>Alt</b> + <b>w</b>
 MAIN_TOOLTIP_BACKCOLOR0;Hintergrundfarbe der Vorschau basierend auf dem <b>Oberflächendesign</b>.\nTaste: <b>9</b>
 MAIN_TOOLTIP_BACKCOLOR1;Hintergrundfarbe der Vorschau: <b>Schwarz\n</b>Taste: <b>9</b>
 MAIN_TOOLTIP_BACKCOLOR2;Hintergrundfarbe der Vorschau: <b>Weiß</b>\nTaste: <b>9</b>
 MAIN_TOOLTIP_BACKCOLOR3;Hintergrundfarbe der Vorschau: <b>Mittleres Grau</b>\nTaste: <b>9</b>
 MAIN_TOOLTIP_BEFOREAFTERLOCK;<b>Vorher-Ansicht:</b> Sperren / Entsperren\n\n<b>Gesperrt</b>: Friert die Vorher-Ansicht ein, so\ndass sich die Gesamtwirkung mehrerer\nBearbeitungsschritte beurteilen lässt.\n\n<b>Entsperrt</b>: Die Vorher-Ansicht hinkt dem\naktuellen Bild immer einen Bearbeitungs-\nschritt hinterher.
 MAIN_TOOLTIP_HIDEHP;Linkes Bedienfeld ein-/ausblenden.\nTaste: <b>l</b>
-MAIN_TOOLTIP_INDCLIPPEDH;Anzeige zu heller Bereiche ein-/ausschalten.\nTaste: <b>&lt;</b>
-MAIN_TOOLTIP_INDCLIPPEDS;Anzeige zu dunkler Bereiche ein-/ausschalten.\nTaste: <b>&gt;</b>
+MAIN_TOOLTIP_INDCLIPPEDH;Anzeige zu heller Bereiche ein-/ausschalten.\nTaste: <b>&lt
```
Comment (Beep6581) [inline rtdata/languages/Deutsch]: @TooWaBoo did you remove the end tag on purpose?

File: rtdata/languages/Deutsch [hunk 2]
```diff
@@ -1835,8 +1881,9 @@ TP_PREPROCESS_PDAFLINESFILTER;PDAF-Zeilenfilter
 TP_PREPROCESS_PDAFLINESFILTER_TOOLTIP;Minimiert Streifenrauschen, das bei Gegenlichtaufnahmen mit\nsichtbaren Flares einiger spiegellosen Sony-Kameras entsteht.
 TP_PRSHARPENING_LABEL;Nach Skalierung schärfen
 TP_PRSHARPENING_TOOLTIP;Schärft das Bild nach der Größenänderung.\nFunktioniert nur mit der Methode “Lanczos“.\nDas Ergebnis wird nicht in RawTherapee\nangezeigt.\n\nWeitere Informationen finden Sie auf “RawPedia“.
-TP_RAWCACORR_AUTO;Automatische Korrektur
+TP_RAWCACORR_AUTO;Autokorrektur
 TP_RAWCACORR_AUTOIT;Iterationen
+TP_RAWCACORR_AUTOIT_TOOLTIP;Diese Einstellung ist verfügbar, wenn "Autokorrektur" aktiviert ist.\nDie Autokorrektur ist konservativ, dh sie korrigiert häufig nicht alle\nchromatischen Aberrationen. Um die verbleibenden chromatischen\nAberrationen zu korrigieren, können Sie bis zu fünf Iterationen\nverwenden. Jede Iteration verringert die verbleibende chromatische\nAberration auf Kosten zusätzlicher Verarbeitungszeit.
```
Comment (heckflosse) [inline rtdata/languages/Deutsch]: dh => d.h.

File: rtdata/languages/Deutsch [hunk 3]
```diff
@@ -2012,7 +2042,7 @@ TP_RETINEX_NEUTRAL;Zurücksetzen
 TP_RETINEX_NEUTRAL_TIP;Setzt alle Regler und Kurven\nauf ihre Standardwerte zurück.
 TP_RETINEX_OFFSET;Ausgleich (Helligkeit)
 TP_RETINEX_SCALES;Gaußscher Gradient
-TP_RETINEX_SCALES_TOOLTIP;Steht der Regler auf 0 sind alle Iterationen identisch.\nBei > 0 werden Skalierung und Radius reduziert,\nbei &lt; 0 erhöht.
+TP_RETINEX_SCALES_TOOLTIP;Steht der Regler auf 0 sind alle Iterationen identisch.\nBei > 0 werden Skalierung und Radius reduziert,\nbei &lt
```
Comment (heckflosse) [inline rtdata/languages/Deutsch]: Fehlt da nicht was hinter dem, `\nbei &lt` oder steht das jetzt an einer anderen Stelle?

**Comment by TooWaBoo** [comment]
I've merged my already done translation with Excel 'cause the deleted keys were not listet at the end of the file in dev. Maybe that something went wrong.

**Comment by TooWaBoo** [comment]
Yes , there went somthing wrong I've to do it again.

**Comment by heckflosse** [comment]
Ok

**Comment by TooWaBoo** [comment]
Should be OK now.

File: rtdata/languages/Deutsch [hunk 4]
```diff
@@ -1101,32 +1083,26 @@ PARTIALPASTE_SOFTLIGHT;Weiches Licht
...
-PREFERENCES_CACHECLEAR_ALLBUTPROFILES;Alle Dateien im Zwischenspeicher löschen mit Ausnahme der Bearbeitungsprofile:
-PREFERENCES_CACHECLEAR_ONLYPROFILES;Alle Bearbeitungsprofile im Zwischenspeicher löschen:
-PREFERENCES_CACHECLEAR_SAFETY;Nur Dateien im Zwischenspeicher werden gelöscht. Bearbeitungsprofile im Bildordner bleiben unberührt.
 PREFERENCES_CACHEMAXENTRIES;Maximale Anzahl der Miniaturbilder im Festplatten-Cache
 PREFERENCES_CACHEOPTS;Einstellungen des Festplatten-Cache für Miniaturbilder
 PREFERENCES_CACHETHUMBHEIGHT;Maximale Höhe der Miniaturbilder
+PREFERENCES_CIEART;CIECAM02-Optimierung
+PREFERENCES_CIEART_FRAME;CIECAM02-spezifische Einstellungen
+PREFERENCES_CIEART_LABEL;Einfache statt doppelte Genauigkeit
+PREFERENCES_CIEART_TOOLTIP;Wenn aktiviert, werden CIECAM02-Berechnungen mit einfacher Genauigkeit statt doppelter durchgeführt. Dadurch wird der Vorgang etwas beschleunigt. Die Qualität nimmt jedoch geringfügig ab.
```
Comment (heckflosse) [inline rtdata/languages/Deutsch]: These are not used anymore since a while. Maybe better to continue tomorrow?

**Comment by TooWaBoo** [comment]
@Beep has deleted some keys and this made the DE locale inconsistent. I combined default with the DE locale with Excel and than this mistake happens because the ";" is not only field a delimiter (I know now 😄 ). The current dev now contains these outdated keys. The file is in sync with dev. So everything is OK from my side.

**Comment by beep** [comment]
(Not me, I don’t think.)

**Comment by TooWaBoo** [comment]
> 
> 
> (Not me, I don’t think.)

Sorry my mistake. 😅

**Comment by TooWaBoo** [comment]
If combined the Deutsch locale with default in this zip-file.
[Deutsch_sync_with_default.zip](https://github.com/Beep6581/RawTherapee/files/2606179/Deutsch_sync_with_default.zip)

**Pull request merged**